backtracking/subset_sum.cpp

```diff
@@ -0,0 +1,106 @@
+/**
+ * @file
+ * @brief Program to Count number of subsets(both continuous and non-continuous
+ * subarrays) with a given sum
```

Comment:
```suggestion
 * @brief Program to Count number of subsets non-continuous
 * subarray with a given sum
```
Your code will fail if only continuous array is allowed.